EVENT.md

```diff
@@ -517,15 +517,16 @@ Trigger registration:
   }
 }
 ```
-If keys in the filters JSON match keys in `filter_data`, the trigger is
-completely ignored if the intersection is empty.
+If keys in the `filters` dictionnary match keys in the `filter_data` dictionnary
+and the intersection of their values is empty, the trigger is ignored.
 
-Note: A key which is present in one JSON and not the other will not be included
-in the matching logic.
+eg. Given a "conversion_subdomain" key present in both `filter_data` and
+`filters` dictionnaries. If the values of the `filters`'s "conversion_subdomain"
+key do not include "electronics.megastore" or "electronics2.megastore", the
+trigger gets ignored.
 
-Note: The filter JSON does not support nested dictionaries or lists.
-`filter_data` and `filters` are only allowed to have a list of values with
-string type.
+Note: A key which is present in one dictionary and not the other will not be
+included in the matching logic (ie. the trigger will be considered).
```

Comment:
Change unrelated to OR'd filters. When first reading on filters, I had to test it to understand how they work.  I'm hoping that this wording makes it a bit clearer. Let me know what you think and/or ifwe'd prefer to have this change in a different PR.

Reply:
Let's make this change in a separate PR.

Reply:
https://github.com/WICG/attribution-reporting-api/pull/636